git-hub

```diff
@@ -1471,6 +1474,143 @@ class PullCmd (IssueCmd):
             git_quiet(1, 'fetch', remote_url, remote_branch)
             git_quiet(1, 'checkout', 'FETCH_HEAD', *args.args)
 
+    class ReviewCmd (PullUtil):
+        cmd_help = "review the pull request"
+        @classmethod
+        def setup_parser(cls, parser):
+            parser.add_argument('pull',
+                help="number identifying the pull request to review")
+            parser.add_argument('-b', '--base', metavar='BASE',
+                default='upstream/master',
+                help="branch (or git ref) from which to start reviewing "
+                    "(defaults to 'upstream/master')")
+            parser.add_argument('-s', '--auto-skip', metavar='N', type=int,
+                help="automatically skip the first N commits")
+        @classmethod
+        def run(cls, parser, args):
+            # First confirm that a diff tool has been configured
+            if not git_config('diff.tool', prefix=''):
+                sys.stdout.write("Use '\033[94m"
+                    "git config diff.tool <your-favourite-diff-tool>\033[0m' "
+                    "to configure a diff tool\n")
+                die("No diff tool has been configured. Aborting.")
+
+            # BUG: using vimdiff as the diff tool hangs the terminal :(
+            # The reason is most likely that vimdiff doesn't open in a new
+            # window altogether. For now, we just reject vimdiff.
+            if git_config('diff.tool', prefix='') == 'vimdiff':
+                die("vimdiff cannot be used as the diff tool because of a "
+                    "known issue (please consider using gvimdiff instead).")
```

Comment:
Add link to issue? Also it would be nice to solve before this gets merged, it doesn't look very good not to support one of the main diff tools out there.